deploy/helm/kubecf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -67,6 +67,18 @@
         exec:
           command: [sh, -c, 'ss -nlt sport = 17019 | grep "LISTEN.*:17019"']
 
+{{if .Values.testing.brain_tests.enabled -}}
+# For brain tests, allow garden access to the insecure registry they will create.
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/insecure_docker_registry_list?
+  value:
+  - "insecure-registry.tcp.((system_domain)):20005"
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties/grootfs?/insecure_docker_registry_list
+  value:
+  - "insecure-registry.tcp.((system_domain)):20005"
```

Comment:
```suggestion
  path: /instance_groups/name=diego-cell/jobs/name=garden/properties/grootfs?/insecure_docker_registry_list/-
  value:
    "insecure-registry.tcp.((system_domain)):20005"
```
One more.

Reply:
Ditto. Yes, 3 places, all the same thing, with slightly different properties.